index.html

```diff
@@ -125,24 +125,24 @@ <h3 class="article__title">
             </div>
           </div>
```

Comment:
this closing `</div>` tag needs to be a `</article>` tag to match the above; this change will fix a lot of the styling issues on the site
![image](https://user-images.githubusercontent.com/49273611/199187164-34174bf8-3ee6-44ae-8931-af00546798ba.png)
![image](https://user-images.githubusercontent.com/49273611/199187205-0dfed501-109e-4ccb-8d2b-2f1758d7d975.png)